### PR TITLE
chm-391 add national target report view page

### PR DESCRIPTION
**Pull request by liyuanmontreal**
### General description
migrate National Target report page

### Designs
refer to absch page

image

### CODE REVIEW

**Review by blaisefonseca — APPROVED** [review]

File: app/app-text/views/reports/chm/national-target.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+    "generalInformation": "General information",
+    "country": "Country",    
+    "levelOfApplication": "Level of application",
+    "jurisdiction": "Jurisdiction",
+    "relevanceOfNationalTargetsToAichiTargets":"Relevance of National Targets to Aichi Targets", 
+    "aichiTargetComponents": "Aichi Target components",
+    "SubAichiTargetsOrTargetComponents": "Sub-Aichi Targets  or Target components",
+    "rationaleForTheNationalTarget" :"Rationale for the National Target",
+    "relevantDocumentsAndInformation":"Relevant documents and information",
+    "otherRelevantWebsiteAddressOrAttachedDocuments":"Other relevant website address or attached documents"
```
[the remark this answers is not in the document]
Comment (liyuanmontreal) [reply app/app-text/views/reports/chm/national-target.json]: done

File: app/app-text/views/reports/chm/national-target.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+    "generalInformation": "General information",
+    "country": "Country",   
+    "title":"Title", 
+    "level": "Level of application",
+    "jurisdiction": "Jurisdiction",
+    "details":"Details on the level of application",
+    "relevance":"Relevance of National Targets to Aichi Targets", 
+    "aichiTarget": "Aichi Target",
+    "aichiTargetComponents": "Aichi Target components",
+    "SubAichiTargets": "Sub-Aichi Targets",
```
Comment (tahmeedshah) [inline app/app-text/views/reports/chm/national-target.json:11]: 'SubAichiTargets' use camelCase

File: app/views/forms/view/chm/national-target.vue [hunk 3]
```diff
@@ -0,0 +1,105 @@
+<template> 
+     <div id="Record" class="record ">
+        <div class="record-body  bg-white" v-if="document">
+
+            <section>
+                <div  v-if="document.government">                          
+                    <legend>{{ t("generalInformation") }} </legend>               
+                    <div v-if="document.government">
```
Comment (tahmeedshah) [inline app/views/forms/view/chm/national-target.vue:8]: v-if="document.government" is not required here if the parent div has the same condition.

Comment (liyuanmontreal) [reply app/views/forms/view/chm/national-target.vue:8]: fix , change outside condition to display if one of three fields exist.

File: app/app-text/views/reports/chm/national-target.json (hunk 2 above)
Comment (liyuanmontreal) [reply app/app-text/views/reports/chm/national-target.json:11]: fixed

**Review by tahmeedshah — APPROVED** [review]

File: app/views/forms/view/chm/national-target.vue [hunk 4]
```diff
@@ -0,0 +1,104 @@
...
+
+                        <div class="km-value">                    
+                            <div  v-for="target in document.aichiTargets" >                    
+                                <km-term :value="target" :locale="locale" ></km-term>                
+                            </div>
+                        </div>
+                    </div> 
+        
+                    <div v-if="document.otherAichiTargets">     
+                        <label>{{ t("SubAichiTargets") }} </label>
```
Comment (blaisefonseca) [inline app/views/forms/view/chm/national-target.vue:64]: wrong key 
```suggestion
                        <label>{{ t("subAichiTargets") }} </label>      
```

Comment (liyuanmontreal) [reply app/views/forms/view/chm/national-target.vue]: fixed

**Review by blaisefonseca — APPROVED** [review]

**Review by blaisefonseca — APPROVED** [review]

**Pull request merged**